sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/CancelTranslationTests.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.translation.document;
+
+import com.azure.ai.translation.document.models.BatchRequest;
+import com.azure.ai.translation.document.models.SourceInput;
+import com.azure.ai.translation.document.models.TargetInput;
+import com.azure.ai.translation.document.models.TranslationStatus;
+import com.azure.core.test.annotation.RecordWithoutRequestBody;
+import com.azure.core.util.polling.SyncPoller;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CancelTranslationTests extends DocumentTranslationClientTestBase {
+
+    @RecordWithoutRequestBody
+    @Test
+    public void testCancelTranslation() {
+
+        String sourceUrl = createSourceContainer(ONE_TEST_DOCUMENTS);
+        SourceInput sourceInput = TestHelper.createSourceInput(sourceUrl, null, null, null);
+
+        String targetUrl = createTargetContainer(null);
+        String targetLanguageCode = "fr";
+        TargetInput targetInput = TestHelper.createTargetInput(targetUrl, targetLanguageCode, null, null, null);
+        List<TargetInput> targetInputs = new ArrayList<>();
+        targetInputs.add(targetInput);
+        BatchRequest batchRequest = new BatchRequest(sourceInput, targetInputs);
+        SyncPoller<TranslationStatus, Void> poller = getDocumentTranslationClient()
```

Comment:
Replace the multiple calls to `getDocumentTranslationClient()` with a single creation and use that object in the other places.

`DocumentTranslationClient documentTranslationClient = getDocumentTranslationClient();`

Reply:
Here and other tests.

Reply:
Updated.